src/utils.py

```diff
@@ -353,6 +359,13 @@ def write_node_key_file(key):
     sp.run(['chmod', '0600', c.NODE_KEY_FILE], check=False)
 
 
+def generate_node_key():
+    command = [c.BINARY_FILE, 'key', 'generate-node-key', '--file', c.NODE_KEY_FILE]
```

Comment:
yeah, from the PR description
`check=False, so for binaries that don't have the key argument it will only throw a Warning in the debug-log. I think it mostly exist. I found one that did not have it, Enjin Matrix.`